### PR TITLE
[civ2][lint/2] migrate code format to civ02

### PR DESCRIPTION
**Pull request by can-anyscale**
Migrate code format to civ02. This is one of the lint that relies on most of the pip packages in lint-requirements.txt

Test: CI

### CODE REVIEW

File: ci/forge/Dockerfile.forge [hunk 1]
```diff
@@ -31,7 +31,8 @@ curl -fsSL https://deb.nodesource.com/setup_18.x | bash -
 
 apt-get update
 apt-get install -y \
-  awscli docker-ce-cli nodejs build-essential python-is-python3 python3-pip
+  awscli docker-ce-cli nodejs build-essential python-is-python3 \
+  python3-pip openjdk-8-jre wget
```
Comment (aslonnie) [inline ci/forge/Dockerfile.forge:35]: which part needs jre?

I am not particularly happy with the jre installation.. but I guess I just should take it as it is (much) better than running on the ci test image..

Comment (can-anyscale) [reply ci/forge/Dockerfile.forge:35]: there is this lint written in java: https://github.com/ray-project/ray/blob/master/ci/lint/format.sh#L113

File: ci/forge/Dockerfile.forge [hunk 2]
```diff
@@ -1,4 +1,5 @@
 # syntax=docker/dockerfile:1.3-labs
+# shellcheck disable=SC2148
```
Comment (aslonnie) [inline ci/forge/Dockerfile.forge:2]: revert this line?

File: ci/forge/Dockerfile.forge [hunk 3]
```diff
@@ -30,7 +31,8 @@ curl -fsSL https://deb.nodesource.com/setup_18.x | bash -
 
 apt-get update
 apt-get install -y \
-  awscli docker-ce-cli nodejs build-essential python-is-python3 python3-pip
+  awscli docker-ce-cli nodejs build-essential python-is-python3 \
+  python3-pip openjdk-8-jre wget
```
Comment (aslonnie) [inline ci/forge/Dockerfile.forge:35]: do we need to install shellcheck somewhere?

how do we know that we are not missing any checks due to missing tools/dependencies?

Comment (can-anyscale) [reply ci/forge/Dockerfile.forge:35]: shellcheck is a part of the python/requirements/lint-requirements.txt file

Comment (aslonnie) [reply ci/forge/Dockerfile.forge:35]: maybe consider change the script so that on ci it will fail when the required commands are missing.

Comment (aslonnie) [reply ci/forge/Dockerfile.forge:35]: maybe as a TODO.

Comment (can-anyscale) [reply ci/forge/Dockerfile.forge:34]: TODO added to hard fail on the missing of lint tools

**Comment by can-anyscale** [comment]
Also need a stamp from @edoakes , @ericl  or @richard4912 on the lint changes. Thankks

**Review by aslonnie — APPROVED** [review]

**Comment by can-anyscale** [comment]
Someone just broke this lint on master lol

**Comment by aslonnie** [comment]
break speed > fix speed

**Review by edoakes — APPROVED** [review]

**Comment by can-anyscale** [comment]
It's greeeen now: https://buildkite.com/ray-project/premerge/builds/843#_

**Pull request merged**